Description: Change XSane's GIMP menu layout
 Group XSane actions in their own submenu in the Acquire menu.
Author: Julien BLACHE <jblache@debian.org>
Forwarded: not-needed

Index: xsane-0.998/src/xsane-text.h
===================================================================
--- xsane-0.998.orig/src/xsane-text.h	2002-08-16 10:34:58.000000000 +0200
+++ xsane-0.998/src/xsane-text.h	2011-02-04 19:50:36.505016000 +0100
@@ -870,8 +870,8 @@
 #define XSANE_GIMP_INSTALL_HELP		_("This function provides access to scanners and other image acquisition devices through the SANE (Scanner Access Now Easy) interface.")
 
 /* Menu path must not be translated, this is done by the gimp. Only translate the text behind the last "/" */
-#define XSANE_GIMP_MENU_DIALOG		_("<Toolbox>/File/Acquire/XSane: Device dialog...")
-#define XSANE_GIMP_MENU			_("<Toolbox>/File/Acquire/XSane: ")
+#define XSANE_GIMP_MENU_DIALOG		_("<Toolbox>/File/Acquire/XSane/Device dialog...")
+#define XSANE_GIMP_MENU			_("<Toolbox>/File/Acquire/XSane/")
 #define XSANE_GIMP_MENU_DIALOG_OLD	_("<Toolbox>/Xtns/XSane/Device dialog...")
 #define XSANE_GIMP_MENU_OLD		_("<Toolbox>/Xtns/XSane/")
 
